.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+name: "CodeQL"
+
+on:
+  push:
+    branches: ['*master', '*0.[0-9].x']
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: ['*master', '*0.[0-9].x']
+  schedule:
+    - cron: '0 11 * * 3'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-18.04
+
+    strategy:
+      fail-fast: false
+      matrix:
+        # Override automatic language detection by changing the below list
+        # Supported options are ['csharp', 'cpp', 'go', 'java', 'javascript', 'python']
+        language: ['cpp', 'python']
```

Comment:
Do we need to specify "C" here too? (Or should we?)

Reply:
I'm not sure if CodeQL supports C per se. At any rate, that would only be for the code security analysis part. Building the code should take place regardless.